rero_ils/modules/holdings/api.py

```diff
@@ -318,6 +404,13 @@ def create_holding(
     }
     if electronic_location:
         data['electronic_location'] = [electronic_location]
+    if holdings_type:
+        holdings_type = holdings_type
+    else:
+        holdings_type = 'monograph'
+    data['holdings_type'] = holdings_type
```

Comment:
Not mandatory, but in less code:
```python
holdings_type_value = 'monograph'
if holdings_type:
    holdings_type_value = holdings_type
data['holdings_type'] = holdings_type_value
```
Think about next lines that use holdings_type.